content/spin/v3/faq.md

```diff
@@ -0,0 +1,42 @@
+title = "Frequently Asked Questions"
+template = "spin_main"
+date = "2023-11-04T00:00:01Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/v3/whats-new.md"
```

Comment:
This url string doesn't match the actual `.../faq` url.  Maybe it doesn't matter?  But something I noted.

Reply:
Ugh, yes.  That's historic.  Thanks for the spot.  Fixed